Doc/tutorial/controlflow.rst

```diff
@@ -306,6 +306,9 @@ If you are using classes to structure your data
 you can use the class name followed by an argument list resembling a
 constructor, but with the ability to capture attributes into variables::
 
+    from dataclasses import dataclass
+    @dataclass
+
```

Comment:
The `@dataclass` line should be directly abouve the `class Point:` line.

Reply:
fixed